tests/test_demand.py

```diff
@@ -123,6 +136,16 @@ def test_calculate_heat_demand(self):
 
         assert a["kWh"].sum() == 10969639.113628691
 
+    def test_adjust_heat_demand(self):
+
+        result = adjust_heat_demand(
+            temperature=self.weather["temp_air"],
+            heating_limit_temp=self.heating_lim_temp,
+            demand=self.heat_load["kWh"],
+        )
+
+        assert result.sum() == 10969639.113628691
```

Comment:
```suggestion
        assert result.sum() == heat_load["kWh"].sum()
```
I think here you test, that the sum stays the same (because the load from summer times is distributed to the rest of the year). Could you also write a test that checks that expected values are set to zero please?

Reply:
Good idea! :)

Done with commit https://github.com/greco-project/pvcompare/pull/137/commits/289894fd4fa37b858f95664c1c21b6d0bda90064